cmake/common.cmake

```diff
@@ -558,17 +558,16 @@ set_property(TARGET http_parser.host PROPERTY POSITION_INDEPENDENT_CODE ON)
 
 # Common test args for Python scripts starting up CCF networks
 if(PBFT)
-  set(PBFT_ARG "--pbft")
+  set(CONSENSUS_ARG "pbft")
 else()
-  unset(PBFT_ARG)
+  set(CONSENSUS_ARG "raft")
 endif()
 
 set(CCF_NETWORK_TEST_ARGS
   ${TEST_IGNORE_QUOTE}
   ${TEST_ENCLAVE_TYPE}
   -l ${TEST_HOST_LOGGING_LEVEL}
   -g ${CCF_DIR}/src/runtime_config/gov.lua
-  ${PBFT_ARG}
```

Comment:
Why was this moved out and duplicated?

Reply:
It's not the same argument. I am passing `--consensus pbft/raft` instead of `--pbft`. But I did move it back into `CCF_NETWORK_TEST_ARGS`